packages/ali/src/lib/AliTypes.ts

```diff
@@ -0,0 +1,245 @@
+/*
+ * © 2023 Thoughtworks, Inc.
+ */
+
+import { COMPUTE_PROCESSOR_TYPES } from '@cloud-carbon-footprint/core'
+
+export const SPECIFICATION_FAMILY_COMPUTE_PROCESSOR_MAPPING: {
+  [series: string]: string[]
+} = {
+  'ecs.g8ae': [COMPUTE_PROCESSOR_TYPES.AMD_EPYC_1ST_GEN],
+  'ecs.g7se': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.g7a': [COMPUTE_PROCESSOR_TYPES.AMD_EPYC_1ST_GEN],
+  'ecs.g7': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.g7t': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.g7nex': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.g6t': [COMPUTE_PROCESSOR_TYPES.CASCADE_LAKE],
+  'ecs.g6e': [COMPUTE_PROCESSOR_TYPES.CASCADE_LAKE],
+  'ecs.g6': [COMPUTE_PROCESSOR_TYPES.CASCADE_LAKE],
+  'ecs.sn2ne': [COMPUTE_PROCESSOR_TYPES.SKYLAKE],
+  'ecs.g5': [
+    COMPUTE_PROCESSOR_TYPES.CASCADE_LAKE,
+    COMPUTE_PROCESSOR_TYPES.SKYLAKE,
+  ],
+  'ecs.g5ne': [
+    COMPUTE_PROCESSOR_TYPES.CASCADE_LAKE,
+    COMPUTE_PROCESSOR_TYPES.SKYLAKE,
+  ],
+  'ecs.c8ae': [COMPUTE_PROCESSOR_TYPES.AMD_EPYC_1ST_GEN],
+  'ecs.c7re': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.c7se': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.c7nex': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.c7': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
+  'ecs.c7t': [COMPUTE_PROCESSOR_TYPES.ICELAKE],
```

Comment:
I noticed that ICELAKE is missing from the `AliFootprintEstimationConstants.ts` file. Should this be added there to support this mapping?